recipes/sbp/all/conandata.yml

```
@@ -6,3 +6,10 @@ sources:
     sha256:
       source: f2fb738f49112b25e7849ca0c75415159127e9b5373b13e7027362b8fa0b1224
       cmake: 6077d2a754d013e3cb9826f589e47b19ab01f4d91ede4f5bfc14db74bc5dc894
+  "4.2.0":
+    url:
+      source: "https://github.com/swift-nav/libsbp/archive/refs/tags/v4.2.0.tar.gz"
+      cmake: "https://github.com/swift-nav/cmake/archive/31604e72e72c09fa32effdbc37acc79dda7c99d7.zip"
+    sha256:
+      source: 9ee9808394867405938505fb0aa52ffeb8d98b7ce222e47629ffabdc9e23d3e4
+      cmake: 22e0ef6915ad5accfa6c0b30cb75a7d409ee1e08746f5ee92311015aa3826246
```

```suggestion
      source: "f2fb738f49112b25e7849ca0c75415159127e9b5373b13e7027362b8fa0b1224"
      cmake: "6077d2a754d013e3cb9826f589e47b19ab01f4d91ede4f5bfc14db74bc5dc894"
  "4.2.0":
    url:
      source: "https://github.com/swift-nav/libsbp/archive/refs/tags/v4.2.0.tar.gz"
      cmake: "https://github.com/swift-nav/cmake/archive/31604e72e72c09fa32effdbc37acc79dda7c99d7.zip"
    sha256:
      source: "9ee9808394867405938505fb0aa52ffeb8d98b7ce222e47629ffabdc9e23d3e4"
      cmake: "22e0ef6915ad5accfa6c0b30cb75a7d409ee1e08746f5ee92311015aa3826246"
```
Keep values as string

---

Thanks! I've tried this, but it still complained because of the latest changes to the `conan-center` hook. It seems one of the checks ignores the whole `url` dict stuff. But I was able to rewrite the structure here a bit and it works.

---

I did those changes, so if something is broken, it's my fault! 😅 
We want to avoid empty values and exotic key names, but some recipes will not fit those rules due their nature. We can add this package as exception and skip that validation. We do it for other hooks usually, nothing uncommon.

---

No worries, all good! I think the new structure makes it easier to parse in the conanfile and is also used similarly in other recipes. :)

---

Great!